compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/scene/ComposeSceneMediator.uikit.kt

```diff
@@ -563,6 +636,11 @@ internal class ComposeSceneMediator(
             || scene.sendKeyEvent(keyEvent)
             || onKeyEvent(keyEvent)
 
+
+    companion object {
+        private val FOCUS_CHANGE_ANIMATION_DURATION = 0.15.seconds
```

Comment:
we usually have such consts in file-private scope, without `companion object`

Reply:
Done